lapis2/build.gradle

```diff
@@ -51,3 +53,18 @@ tasks.named('test') {
 tasks.named('bootBuildImage') {
     imageName = "ghcr.io/genspectrum/${project.name}"
 }
+
+// workaround related to https://github.com/springdoc/springdoc-openapi-gradle-plugin/issues/102
```

Comment:
This can hopefully be removed when https://github.com/springdoc/springdoc-openapi-gradle-plugin/issues/102 is resolved.